cmd/dbc/auth.go

```diff
@@ -190,13 +194,22 @@ func (m loginModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				}
 			})
 	case auth.Credential:
-		return m, tea.Sequence(func() tea.Msg {
+		return m, func() tea.Msg {
 			if err := auth.AddCredential(msg, true); err != nil {
 				return err
 			}
-			return nil
-		}, tea.Println("Authentication successful!"),
-			tea.Quit)
+			return authSuccessMsg{cred: msg}
+		}
+	case authSuccessMsg:
+		return m, tea.Sequence(tea.Println("Authentication successful!"),
+			func() tea.Msg {
+				if auth.IsColumnarPrivateRegistry((*url.URL)(&msg.cred.RegistryURL)) {
+					return auth.FetchColumnarLicense(&msg.cred)
+				}
+				return nil
+			})
+	case nil:
+		return m, tea.Quit
```

Comment:
This looked fine to me but my auth command hung indefinitely. I think bubbletea doesn't like nil being used a signal.
```suggestion
				return tea.Quit()
			})
```